<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>telaCadastroUser</class>
 <widget class="QDialog" name="telaCadastroUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>1</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>600</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>450</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BIBLIOTECA</string>
  </property>
  <property name="windowIcon">
   <iconset resource="arquivos.qrc">
    <normaloff>:/images/images/logo.ico</normaloff>:/images/images/logo.ico</iconset>
  </property>
  <widget class="QPushButton" name="cadastrarFuncionario_btn">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>520</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{font-size: 20px;
border-radius: 5px;
background-color:#2a428c;
color: #fefefe;
font-family:Franklin Gothic Heavy;}

:hover{
background-color:#3c58a0;}</string>
   </property>
   <property name="text">
    <string>CADASTRAR</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="arquivos.qrc">:/images/images/888.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{font-size: 13px;
border-radius: 5px;
background-color:#2a428c;
color: #fefefe;
font-family:Franklin Gothic Heavy;}

:hover{
background-color:#3c58a0;}
</string>
   </property>
   <property name="text">
    <string>&lt; Voltar</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>271</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">color: #fefefe;
font-family:Franklin Gothic Heavy;
font-size:13px;</string>
      </property>
      <property name="text">
       <string>NOME</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="nomeFuncionario_txt">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid grey;
  border-radius: 5px;
background-color:#fefefe;
color:#303030</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">color: #fefefe;
font-family:Franklin Gothic Heavy;
font-size:13px;</string>
      </property>
      <property name="text">
       <string>TELEFONE</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="telefoneFuncionario_txt">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid grey;
  border-radius: 5px;
background-color:#fefefe;
color:#303030</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">color: #fefefe;
font-family:Franklin Gothic Heavy;
font-size:13px;</string>
      </property>
      <property name="text">
       <string>E-MAIL</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="emailFuncionario_txt">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid grey;
  border-radius: 5px;
background-color:#fefefe;
color:#303030</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="styleSheet">
       <string notr="true">color: #fefefe;
font-family:Franklin Gothic Heavy;
font-size:13px;</string>
      </property>
      <property name="text">
       <string>SENHA</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="senhaFuncionario_txt">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid grey;
  border-radius: 5px;
background-color:#fefefe;
color:#303030</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="styleSheet">
       <string notr="true">color: #fefefe;
font-family:Franklin Gothic Heavy;
font-size:13px;</string>
      </property>
      <property name="text">
       <string>CONFIRMAR SENHA</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="senhaFuncionarioConfirma_txt">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid grey;
  border-radius: 5px;
background-color:#fefefe;
color:#303030</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>451</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#2a428c;
color:#fefefe;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="login_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family:Franklin Gothic Heavy;
font-size:25px;
color:#fefefe;
</string>
    </property>
    <property name="text">
     <string>CADASTRAR FUNCION√ÅRIO</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextSelectableByMouse</set>
    </property>
   </widget>
  </widget>
  <zorder>frame</zorder>
  <zorder>cadastrarFuncionario_btn</zorder>
  <zorder>label_2</zorder>
  <zorder>pushButton</zorder>
  <zorder>layoutWidget</zorder>
 </widget>
 <tabstops>
  <tabstop>nomeFuncionario_txt</tabstop>
  <tabstop>telefoneFuncionario_txt</tabstop>
  <tabstop>emailFuncionario_txt</tabstop>
  <tabstop>senhaFuncionario_txt</tabstop>
  <tabstop>senhaFuncionarioConfirma_txt</tabstop>
 </tabstops>
 <resources>
  <include location="arquivos.qrc"/>
 </resources>
 <connections/>
</ui>
